parm/config/config.base.emc.dyn

```diff
@@ -346,6 +332,28 @@ if [ $DOHYBVAR = "YES" ]; then
     else
         export FHOUT_ENKF=3
     fi
+else
+  export DOIAU=${DOIAU:-"NO"}   # 3DVAR defaults to no IAU
+fi
+
+# IAU related parameters
+if [[ $DOHYBVAR == "NO" && $DOIAU == "YES"  ]]; then
+  export IAUFHRS="6"
+  export IAU_FHROT="3"
+  export IAU_FILTER_INCREMENTS=.true.
+else
+  export IAUFHRS="3,6,9" # Enable 4DIAU or control with 3 increments
+  export IAU_FHROT=$(echo $IAUFHRS | cut -c1)
+fi
+export IAU_DELTHRS=6
+export IAU_OFFSET=6
+export DOIAU_ENKF=${DOIAU:-"YES"}   # Enable 4DIAU for EnKF ensemble
+export IAUFHRS_ENKF="3,6,9"
+export IAU_DELTHRS_ENKF=6
```

Comment:
These should be wrapped in an if-block such as:
```
if [[ $DOHYBVAR = "YES" ]]; then
    export DOIAU_ENKF=${DOIAU:-"YES"}   # Enable 4DIAU for EnKF ensemble.
    export IAUFHRS_ENKF="3,6,9"
    export IAU_DELTHRS_ENKF=6
fi
```

Reply:
updated

Reply:
This IF block is not sufficient.  The IAU*ENKF variables should only be set to these values if DOIAU_ENKF="YES" and for 4D.